images/universal/training/README.md

```
@@ -56,6 +56,7 @@ The image has **two layers** of dependencies:
 1. **Base Image Dependencies** (from `workbench-images`)
    - JupyterLab, notebook extensions, authentication
    - Pre-installed in the base image via its own `pylock.toml`
+   - Latest image can be found [here](https://github.com/opendatahub-io/notebooks/blob/main/manifests/base/params-latest.env#L2)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)